components/server/src/workspace/image-source-provider.ts

```diff
@@ -37,7 +37,7 @@ export class ImageSourceProvider {
 
             const imgcfg = config.image;
             if (ExternalImageConfigFile.is(imgcfg)) {
-                // we're asked to pull the Dockerfile from a repo possibly different than the one we're opening a workspace for (e.g. definitely-gp).
+                // we're asked to pull the Dockerfile from a repo possibly different than the one we're opening a workspace for.
```

Comment:
This has to stay, if I remember correctly.
'Just paying attention to what could break.

Reply:
yes, it was a reason why it broke last time, we create ExternalImageConfigFile above, did not process it here